tools/api-client/python/lib/bmapi.py

```diff
@@ -3,19 +3,47 @@
 # The BMClient class provides one function for each API method,
 # which is called with the arguments to be passed to that method,
 # and returns the JSON response to the method invocation.
-import ConfigParser
-import cookielib
+
+### Imports
+
+# Import stuff from the future.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+# Import regular stuff.
+
 import json
 import os
-import urllib
-import urllib2
+
+# Import Python version specific stuff.
+
+try:
+  import configparser
+except ImportError:
+  import ConfigParser as configparser
+
+try:
+  from http.cookiejar import LWPCookieJar
+except ImportError:
+  from cookielib import LWPCookieJar
+
+try:
+  from urllib.parse import urlparse, urlencode
+  from urllib.request import urlopen, Request, HTTPCookieProcessor, build_opener, install_opener
+  from urllib.error import HTTPError
+except ImportError:
+  from urlparse import urlparse
+  from urllib import urlencode
+  from urllib2 import urlopen, Request, HTTPError, HTTPCookieProcessor, build_opener, install_opener
```

Comment:
Let's maybe be safe and make this a separate try/except block for each different import, rather than assuming they'll all succeed or fail identically?  Or is this definitely risk-free?